tests/run-tests

```diff
@@ -7,37 +7,58 @@ import platform
 import argparse
 import inspect
 import re
+import shutil
+import tempfile
 from glob import glob
 
+# This is the path containing run-tests (i.e. the absolute path of micropython/tests).
 # See stackoverflow.com/questions/2632199: __file__ nor sys.argv[0]
 # are guaranteed to always work, this one should though.
 BASEPATH = os.path.dirname(os.path.abspath(inspect.getsourcefile(lambda: None)))
 
+
+# Convert a path relative to micropython/tests to an absolute path.
 def base_path(*p):
-    return os.path.abspath(os.path.join(BASEPATH, *p)).replace('\\', '/')
+    return os.path.abspath(os.path.join(BASEPATH, *p)).replace("\\", "/")
+
```

Comment:
Did you add these, or `black`? (just wondering)

Reply:
`black` -- The first commit is entirely black's changes (nothing from me).